en/services/parachute.md

```diff
@@ -0,0 +1,165 @@
+| 10  | 1024     | `PARACHUTE_ERROR_FLAGS_GS_CONNECTION_ERROR`    | There is an error with the connection between parachute and Ground Station (GS).                                           |
+| 11  | 2048     | `PARACHUTE_ERROR_FLAGS_GPS_ERROR`              | There is an error with the parachute's GPS.                                                                                |
+| 12  | 4096     | `PARACHUTE_ERROR_FLAGS_SUBSYSTEM_CONNECTION_ERROR` | There is an error with the connection between parachute and subsystem (e.g. remote controller, expansion board, etc.). |
+| 13  | 8192     | `PARACHUTE_ERROR_FLAGS_SUBSYSTEM_FW_ERROR`     | There is an error with the parachute subsystem firmware (e.g. wrong firmware version).                                     |
+| 14  | 16384    | `PARACHUTE_ERROR_FLAGS_RESERVED_1`             | Reserved for future use.                                                                                                   |
+| 15  | 32768    | `PARACHUTE_ERROR_FLAGS_RESERVED_2`             | Reserved for future use.                                                                                                   |
+| 16  | 65536    | `PARACHUTE_ERROR_FLAGS_LOGGING_ERROR`          | There is an error with the parachute's internal logging system.                                                            |
+| 17  | 131072   | `PARACHUTE_ERROR_FLAGS_MODULE_RETIRED`         | This parachute module is retired (i.e. too many deployments).                                                              |
+| 18  | 262144   | `PARACHUTE_ERROR_FLAGS_GLOW_WIRE_ERROR`        | There is an error with the parachute glow wire.                                                                            |
+| 19  | 524288   | `PARACHUTE_ERROR_FLAGS_OFFBOARD_CONNECTION_ERROR` | There is an error with the MAVLink connection between parachute and offboard computer.                                 |
+| 20  | 1048576  | `PARACHUTE_ERROR_FLAGS_IMU_CALIBRATION_ERROR`  | Parachute's internal IMU calibration failed.                                                                               |
+                                                                    |
+
+[PARACHUTE_ERROR_FLAGS_IMU_ERROR](#PARACHUTE_ERROR_FLAGS_IMU_ERROR) is typically used to indicate a hardware failure of the onboard IMU, and [PARACHUTE_ERROR_FLAGS_IMU_CALIBRATION_ERROR](#PARACHUTE_ERROR_FLAGS_IMU_CALIBRATION_ERROR) may indicate that IMU has no calibration or needs re-calibration.
+
+Depending on the parachute product, not all errors may be applicable. For example, [PARACHUTE_ERROR_FLAGS_LOW_POWER](#PARACHUTE_ERROR_FLAGS_LOW_POWER) would be used if there was an additional power source within the parachute module itself that has its power level monitored. 
+
+#### MAV_CMD_SET_PARACHUTE_ARM
```

Comment:
```suggestion
### MAV_CMD_SET_PARACHUTE_ARM
```

Reply:
Oh I see are these all things that can contribute to the different fields of `PARACHUTE_STATUS`? `MAV_CMD_SET_PARACHUTE_ARM` seems more to be on the same level as `PARACHUTE_STATUS` while `PARACHUTE_ERROR_FLAGS`, `PARACHUTE_TRIGGER_FLAGS`, `PARACHUTE_DEPLOYMENT_TRIGGER`, and `PARACHUTE_SAFETY_FLAGS` are on a separate level. But however you want to structure this, either way works

Reply:
you are correct and i think ill just update this one!!!